### DISCUSSION TITLE
IPv6 Leak Protection

### DISCUSSION THREAD

**Discussion by mihaibarbulescu** [vpnhood/VpnHood#42] [Ideas]
Please consider implementing IPv6 leak protection. As far as I know, this could be done by effectively rerouting all IPv6 traffic to a “black hole” within the client device. Alternatively, the capability on server-side to tunnel IPv6 would be awesome. Thank you!

**Comment by trudyhood** [comment]
Yes, There are other priorities than adding IPv6 support but dropping IpV6 packet is easy and must be done.

**Reply by mihaibarbulescu** [reply]
> Yes, There are other priorities than adding IPv6 support but dropping IpV6 packet is easy and must be done.

Indeed. Adding IPv6 support may be cumbersome and not a priority when it comes to server-side. Dropping the packets client-side may be fast and efficient enough to start with.

@trudyhood  would you agree to move this to Issues -> Enhancements?

Thank you!

**Reply by trudyhood** [reply]
> @trudyhood would you agree to move this to Issues -> Enhancements?
Sorry, I forgot to tell you that I have already created an issue for it.
Issue: #43

**Comment by trudyhood** [comment]
Dear @mihaibarbulescu 

We added the ip6 leaking protection in the latest release; I hope it works as you expected. Also, we have done much work to support IPv6, but still, some pieces need to be done to fully support IPv6, and we postponed it for later.

Your idea, indeed, increased its priority. Thank you for your contribution.

**Comment by malikshi** [comment]
isn't time to try vpnhood with ipv6 support?

**Reply by trudyhood** [reply]
Many progress for IPv6 have been done, but why do you see it should have priority?

**Comment by trudyhood** [comment]
Dear @malikshi,
want to announce to you that IPv6 support is ready and we have published it right now. You make us happy if you tell us any Ideas or see any issue in VpnHood.

# IPv6 Support
VpnHood supports IPv6, but in Windows, you need to be already on the IPv6 network, so all your network traffic will be tunneled through VpnHood. In Windows, you will not have access to IPv6 sites if your network is not already configured to use IPv6. In Android, however, you will get an Ipv6 address and access to all IPv6 sites regardless of your network configuration.
